crates/re_renderer/shader/decodings.wgsl

```diff
@@ -1,41 +1,46 @@
 #import <./types.wgsl>
 
 
-/// Loads an RGBA texel from a texture holding an NV12 encoded image at the given screen space coordinates.
-fn decode_nv12(texture: texture_2d<u32>, coords: vec2i) -> vec4f {
+/// Loads an RGBA texel from a texture holding an NV12 or YUY2 encoded image at the given screen space coordinates.
+fn decode_nv12_or_yuy2(sample_type: u32, texture: texture_2d<u32>, coords: vec2i) -> vec4f {
```

Comment:
it surprises me that combining these two into one function did much difference

Reply:
Yeah -- this was the final tweak that got things  to work :-(